pages/explorer/races/[raceId]/units/[unitId].tsx

```diff
@@ -92,11 +95,23 @@ const UnitDetail: NextPage<UnitDetailProps> = ({
   // Obtain the total upkeep cost of the squad.
   const totalUpkeepCost = getSquadTotalUpkeepCost(resolvedSquad, ebpsData);
 
+  const metaKeywords = generateKeywordsString([
+    `${resolvedSquad.ui.screenName} coh3`,
+    `${resolvedSquad.ui.screenName} ${localizedRace}`,
+    `${resolvedSquad.ui.screenName}`,
+    `${resolvedSquad.ui.screenName} ${raceId}`,
+  ]);
+
   return (
     <>
       <Head>
-        <title>{resolvedSquad.ui.screenName} - COH3 Explorer</title>
+        <title>{`${resolvedSquad.ui.screenName} - COH3 Explorer`}</title>
         <meta name="description" content={`${resolvedSquad.ui.screenName} - COH3 Explorer`} />
+        <meta name="keywords" content={metaKeywords} />
+        <meta
+          property="og:image"
+          content={getIconsPathOnCDN(`/icons/${slash(resolvedSquad.ui.iconName)}.png`)}
```

Comment:
When you share link to units, it should show nice image now

Reply:
Awesome! 🚀 didn't know there was another meta property to be used :)